diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter3/LineLength.kt

```diff
@@ -149,12 +150,16 @@ class LineLength(configRules: List<RulesConfig>) : DiktatRule(
                         val textLenAfterFix = node.textLength
                         val blankLinesAfterFix = node.text.lines().size - countCodeLines(node)
 
+                        // checking that any fix may have been made
                         isFixedSmthInChildNode = fixableType !is None
 
+                        // for cases when text doesn't change, and then we need to stop fixes
                         if (textBeforeFix == textAfterFix) {
                             isFixedSmthInChildNode = false
                         }
 
+                        // for cases when fixes don't stop and start generate blank lines due to adding \n at each fix step
```

Comment:
in some kernel cases of long lines, when the limit of line is on last whitespace, we trying to move this right part (which is only whitespace) to the new line. such behavior leads to case, when we generate blank lines time after time
to detect this we count blank lines and make unfix, if necessary

Reply:
Done